ios/Podfile.lock

```diff
@@ -672,4 +672,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 96882cac17249b4e6e01afb5d10b1762a5191341
 
-COCOAPODS: 1.10.0
```

Comment:
Any reason this is changing? Seems like maybe people are using different versions of cocoapods which one is right?

Reply:
Agree, also seem to be downgrading, which I think there would be no reason to.

Reply:
Ah, I keep forgetting to stop committing this. Thanks for catching it